From 8fae371c482877d1f46bd19c5ae97d63dc707148 Mon Sep 17 00:00:00 2001
From: Orit Wasserman <owasserm@redhat.com>
Date: Wed, 9 Oct 2013 10:09:12 +0200
Subject: [PATCH 17/25] migration: add autoconvergence documentation

RH-Author: Orit Wasserman <owasserm@redhat.com>
Message-id: <1381313355-15641-8-git-send-email-owasserm@redhat.com>
Patchwork-id: 54803
O-Subject: [RHEL7.0 qemu-kvm v2 07/10] migration: add autoconvergence documentation
Bugzilla: 921465
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>

From: Juan Quintela <quintela@redhat.com>

This hunk got lost during merge.  It is documentation.

Spotted-by: Peter Lieven <lieven-lists@dlhnet.de>
Signed-off-by: Juan Quintela <quintela@redhat.com>
(cherry picked from commit 9781c3716a4b529a2af64502926a57c11e69e6ca)
---
 qapi-schema.json | 3 +++
 1 file changed, 3 insertions(+)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 qapi-schema.json |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/qapi-schema.json b/qapi-schema.json
index a717fbf..ffe0ed1 100644
--- a/qapi-schema.json
+++ b/qapi-schema.json
@@ -611,6 +611,9 @@
 #          to enable the capability on the source VM. The feature is disabled by
 #          default. (since 1.6)
 #
+# @auto-converge: If enabled, QEMU will automatically throttle down the guest
+#          to speed up convergence of RAM migration. (since 1.6)
+#
 # Since: 1.2
 ##
 { 'enum': 'MigrationCapability',
-- 
1.7.1

